src/main/java/sirpotato/AddCommand.java

```diff
@@ -0,0 +1,21 @@
+package sirpotato;
```

Comment:
It would be nice if the command files are placed in a separate sub-folder in the package